### PR TITLE
View Exams List

### PR DESCRIPTION
**Pull request by Phvanhoang**
image

## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/35519)

## WHAT
- View exams list

## Evidence
image

### CODE REVIEW

**Comment by Phvanhoang** [comment]
Ready

File: app/models/exam.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 class Exam < ApplicationRecord
   belongs_to :subject
+  scope :sort_by_created_at_desc, (lambda do
+    order(created_at: :desc).includes(:subject)
```
Comment (vanvtt-0952) [inline app/models/exam.rb]: `.includes(:subject)` -> viết ở controller dc 

ở đây chỉ cần `scope :sort_by_created_at_desc, -> {order created_at: :desc}`

controller: `@exams = Exam.sort_by_created_at_desc.includes(:subject).paginate(page: par.....`

File: db/seeds.rb [hunk 2]
```diff
@@ -56,3 +56,26 @@
...
+  2.times do |n|
+    Exam.create!(
+      name: Faker::Lorem.sentence(word_count: 5),
+      time: 20,
+      subject_id: sub.id
+    )
+  end
+end
+
+exams_list = Exam.all
```
Comment (vanvtt-0952) [inline db/seeds.rb]: `= Exam.all` thành `= Exam.select :id, :subject_id`

File: db/seeds.rb [hunk 3]
```diff
@@ -56,3 +56,26 @@
     )
   end
 end
+
+subs_list = Subject.all
```
Comment (vanvtt-0952) [inline db/seeds.rb]: `Subject.pluck :id`

**Comment by Phvanhoang** [comment]
Ready

**Pull request merged**